docs/man/rpm.8.scd

```diff
@@ -43,106 +42,70 @@ scripts, file attributes, and descriptive information about the package.
 software to be installed, and source packages, containing the source
 code and recipe necessary to produce binary packages.
 
-One of the following basic modes must be selected: *Query*,
-*Verify*, *Install/Upgrade/Freshen/Reinstall*, *Uninstall*, *Set
-Owners/Groups*, *Show Querytags*, and *Show Configuration*.
-
-# GENERAL OPTIONS
-
-These options can be used in all the different modes.
-
-*-?*, *--help*
-	Print a longer usage message than normal.
-
-*--version*
-	Print a single line containing the version number of *rpm* being
-	used.
-
-*--quiet*
-	Print as little as possible - normally only error messages will be
-	displayed.
-
-*-v*, *--verbose*
-	Print verbose information - normally routine progress messages will
-	be displayed.
-
-*-vv*
-	Print lots of ugly debugging information.
-
-*--rcfile* _FILELIST_
-	Replace the default list of configuration files to be read with
-	_FILELIST_.  See *rpmrc Configuration* for details.
-
-*--load* _FILE_
-	Load an individual macro file.
+# OPERATIONS
+*-e*,
+*--erase*
+	Erase installed packages.
+
+*-i*,
+*--install*
+	Install new package(s) without considering upgrade scenarios.
+	This is considered special usage, normally installations should
+	be done with *--upgrade*.
+
+*-F*,
+*--freshen*
+	Upgrade package(s), but only ones for which an earlier version
+	is installed.
+
+*-q*,
+*--query*
+	Query package files or installed package(s).
+
+*--querytags*
+	Dump known querytags. Useful with the *--queryformat* option.
+
+*--reinstall*
+	Reinstall previously installed package(s).
+
+*--restore*
+	Restore file metadata such as timestamp, owner, group,
+	permissions and capabilities of files of installed package(s).
+
+*-U*,
+*--upgrade*
+	Install or upgrade package(s) to a newer version. Old and/or
+	obsoleted package(s) are removed after the new package is installed.
+
+*-V*,
+*--verify*
+	Verify package(s), comparing information about the installed files in
+	the package with information about the files taken from the package
+	metadata stored in the rpm database. Among other things, verifying
+	compares the size, digest, permissions, type, owner and group of each
+	file. Any discrepancies are displayed.
+
+See *rpm-common*(8) for the operations common to all operations.
```

Comment:
We've got a little recursion here. How about "for the operations common to all rpm executables" :smile: